src/BoilerPlate/App.cpp

```diff
@@ -84,22 +86,65 @@ namespace Engine
 		case SDL_SCANCODE_ESCAPE:
 			OnExit();
 			break;
+		
+		// Move forward with "W" key or with "UP" key
+		
 		case SDL_SCANCODE_UP:
-			ejeYp += 10;
-			ejeYn += 10;
+			inputManager->SetKeyUp(true);
+			SDL_Log("UP was pressed.", keyBoardEvent.keysym.scancode);
 			break;
-		case SDL_SCANCODE_DOWN:
-			ejeYn -= 10;
-			ejeYp -= 10;
+		case SDL_SCANCODE_W:
+			inputManager->SetKeyW(true);
+			SDL_Log("W was pressed.", keyBoardEvent.keysym.scancode);
 			break;
+
+		// ROTATE ship with "A" key or "LEFT" key
+
 		case SDL_SCANCODE_LEFT:
-			ejeXn -= 10;
-			ejeXp -= 10;
+			inputManager->SetKeyLeft(true);
+			SDL_Log("LEFT was pressed.", keyBoardEvent.keysym.scancode);
 			break;
+		case SDL_SCANCODE_A:
+			inputManager->SetKeyA(true);
+			SDL_Log("A was pressed.", keyBoardEvent.keysym.scancode);
+			break;
+
+		// ROTATE ship with "D" key or "RIGHT" key
+
 		case SDL_SCANCODE_RIGHT:
-			ejeXn += 10;
-			ejeXp += 10;
+			inputManager->SetKeyRight(true);
+			SDL_Log("RIGHT was pressed.", keyBoardEvent.keysym.scancode);
+			break;
+		case SDL_SCANCODE_D:
+			inputManager->SetKeyD(true);
+			SDL_Log("D was pressed.", keyBoardEvent.keysym.scancode);
 			break;
+
+		// Creating asteroids with "U" key
+
+		case SDL_SCANCODE_U:
+			inputManager->SetKeyU(true);
+			cout << "Render a Asteroid" << endl;
```

Comment:
If you're logging with `SDL_Log` why use `cout`? Also, we've talked about `using namespace X` before and it's a no no.